Fix SWAP pages

thiagoftsm opened the pull request:
##### Summary
The kernel team changed the SWAP source code [kernel source](https://elixir.bootlin.com/linux/latest/source/mm/page_io.c) when kernel `6.8` was released, so it is necessary to change our code to support this.

##### Test Plan

1. Compile this code on a kernel different of `6.8.x`.
2.  Run this PR with `swap` enabled. You should not have any `libbpf` error.
3. Now compile again on a kernel `6.8.x` (Arch Linux is having it), and run this PR again. You should have the charts.

##### Additional Information
<!-- This is usually used to help others understand your
motivation behind this change. A step-by-step reproduction of the problem is
helpful if there is no related issue. -->

<details> <summary>For users: How does this change affect me?</summary>
  <!--
Describe the PR affects users: 
- Which area of Netdata is affected by the change?
- Can they see the change or is it an under the hood? If they can see it, where?
- How is the user impacted by the change? 
- What are there any benefits of the change? 
-->
</details>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/netdata/netdata?pullRequest=17295) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/netdata/netdata?pullRequest=17295) before we can accept your contribution.<br/><hr/>**Your Name** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/netdata/netdata?pullRequest=17295) it.</sub>

stelfrag commented:
Before PR

```
Apr 01 07:58:53 archlinux netdata[37035]: libbpf: prog 'netdata_swap_readpage_fentry': failed to find kernel BTF type ID of 'swap_readpage': -3
Apr 01 07:58:53 archlinux netdata[37035]: libbpf: prog 'netdata_swap_readpage_fentry': failed to prepare load attributes: -3
Apr 01 07:58:53 archlinux netdata[37035]: libbpf: prog 'netdata_swap_readpage_fentry': failed to load: -3
Apr 01 07:58:53 archlinux netdata[37035]: libbpf: failed to load object 'swap_bpf'
Apr 01 07:58:53 archlinux netdata[37035]: libbpf: failed to load BPF skeleton 'swap_bpf': -3
```

after PR -- no errors

ilyam8 commented:
@thiagoftsm you need to fix your git - see commits author.

thiagoftsm commented:
I fixed guys. Yesterday, my distribution broken `git` after to update many packages, but today the maintainers fixed it. So I was using a VM to work.
I rebased the PR, and it looks like OK now.
